inst/www/shared/shiny_scss/shiny.scss

```diff
@@ -469,3 +469,9 @@ html.autoreload-enabled #shiny-disconnected-overlay.reloading {
   /* override anything bootstrap sets for `.nav` */
   display: none !important;
 }
+
+/* Conditional panels and uiOutput */
+[data-display-if],
+div:where(.shiny-html-output) {
```

Comment:
Can you think of any scenarios this might break existing apps?

Reply:
First and most obvious: this would break apps where users were directly styling the `conditionalPanel()` or `uiOutput()` container using selectors that target our the emitted markup directly. It's hard to know in advance _how_ those apps would be broken.

In most other cases, I believe setting `display: contents` is much closer to what users are expecting happens with the children of the conditional panel or UI outputs. In particular that it's much closer to adding the children directly into the layout.

The original issue in https://github.com/posit-dev/py-shiny/issues/881 is a good example that it's not obvious that both approaches add intermediate containers. Outside of flex/grid layouts the impact of the layout container is minimal, but there's greater impact now that we're using both flex and grid more often.

Reply:
It turns out our shiny application (statnetWeb)[https://github.com/statnet/statnetWeb] is one of those that breaks with this change.

As the current maintainer, and someone not well versed in this syntax, can you pls tell me if there is a straightforward change to the 2 example code chunks below that will update to the new `shiny` syntax required:

```
conditionalPanel(condition = 'input.filetype < 5',
               column(6,
                    br(),
                    fileInput(inputId='rawdatafile', label=NULL, accept='text'),
                    verbatimTextOutput('rawdatafile'))
                )
```

```
             conditionalPanel(condition='input.filetype == 3',
                 column(6,
                        uiOutput('pajchooser'))),
```

Many thanks!

Reply:
Hi @martinamorris, could you please open a new issue? It'd be helpful if you could describe a bit more how the app has broken and how we could reproduce the issue. It's possible that your issue is related to the change in this PR, but it could also be something else -- this PR didn't involve any syntax changes to `conditionalPanel()`.

Reply:
Will do, thx.  The break occurred with the 1.8.1.1 CRAN update, and affects the page rendering for pages constructed with `conditionalPanel`.  I followed the link from your 1.8.1 release tag here, but this may not be the culprit.

Will show how I reproduced the issue, with some screenshots.

Reply:
done:  https://github.com/rstudio/shiny/issues/4099